authtools/backends.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.backends import ModelBackend
+
+
+class CaseInsensitiveEmailBackend(ModelBackend):
+    """
+    This authentication backend assumes that usernames are email addresses and simply lowercases
+    a username before an attempt is made to authenticate said username using Django's ModelBackend.
+
+    Example usage:
+        # In settings.py
+        AUTHENTICATION_BACKENDS = ('authtools.backends.CaseInsensitiveEmailBackend',)
+    """
+    def authenticate(self, username=None, password=None):
+        username = username.lower()
```

Comment:
Also, I think this is going to raise an `AttributeError` that will bubble out of `authenticate` if called like `authenticate(password='foo')`.